lib/honeybadger/splitproject.rb

```diff
@@ -25,13 +25,13 @@ def add_team_specific_notifier(team_postfix, api_key)
         if api_key.nil? || api_key.length == 0
           Honeybadger.instance_eval do
             define_singleton_method("notify_#{team_postfix}") do |exception, options = {}|
-              notify(exception, options)
+              notify_without_team(exception, options)
```

Comment:
I have something I want to discuss with you around this, but am not getting time to write up what I'm thinking with the other stuff going on.

reminder for myself (may or may not be legible to you): "order dependence of other feature-adds with `alias_method` (such as "add current Redshift cluster to message") when the entry point to `Honeybadger.notify` may be either `Honeybadger.notify` or `Honeybadger.notify_#{team}`"